modules/ci_environment/manifests/transition_logs.pp

```diff
@@ -113,4 +113,9 @@
       log_local  => true,
       log_remote => false,
     }
+
+  # S3 backups
+  $backup_dir = hiera('ci_environment::transition_logs::s3_backup_dirs', {})
```

Comment:
I've converted the define to a class since the duplicity job is only declared once. This removes the need for hiera lookup and create_resources.